src/test/scala/cromwell/InvalidRuntimeAttributesSpec.scala

```diff
@@ -20,16 +20,40 @@ class InvalidRuntimeAttributesSpec extends CromwellTestkitSpec with BeforeAndAft
     super.afterAll()
   }
 
-  "A workflow with a task with one invalid runtime attribute" should {
-    "succeed" in {
+  "A workflow with a task with invalid runtime attributes" should {
+    "fail on JES Backend" in {
+      val jesBackend = new JesBackend(actorSystem) {
+        private val anyString = ""
+        private val anyURL: URL = null
+        override lazy val jesConf = new JesAttributes(
+          project = anyString,
+          executionBucket = anyString,
+          endpointUrl = anyURL) {
+        }
+        override def jesUserConnection(workflow: WorkflowDescriptor) = null
+        override lazy val jesCromwellInterface = null
+        override lazy val googleConf = GoogleConfiguration("appName", ServiceAccountMode("accountID", "pem"), None)
+      }
+
+      val workflowSources = WorkflowSourceFiles(SampleWdl.HelloWorld.wdlSource(), SampleWdl.HelloWorld.wdlJson, """ {"jes_gcs_root": "gs://fake/path"} """)
+
+      runWdlWithWorkflowManagerActor(
+        wma = TestActorRef(new WorkflowManagerActor(jesBackend)),
+        sources = workflowSources,
+        stdout = Map.empty,
+        stderr = Map.empty,
+        eventFilter = EventFilter.error(pattern = "Missing required keys in runtime configuration for backend 'JES': docker", occurrences = 1),
+        terminalState = WorkflowFailed
+      )
+    }
+
+    "fail on Local Backend" in {
       runWdl(
         sampleWdl = SampleWdl.HelloWorld,
-        runtime =
-          """ runtime { wrongAttribute: "nop" }""".stripMargin,
-        eventFilter = EventFilter.info(pattern = "transitioning from Running to Succeeded", occurrences = 1),
-        terminalState = WorkflowSucceeded
+        runtime = """ runtime { wrongAttribute: "nop" }""".stripMargin,
+        eventFilter = EventFilter.error(pattern = "Unknown keys found in runtime configuration: wrongAttribute", occurrences = 1),
+        terminalState = WorkflowFailed
```

Comment:
I fixed this in my changes but it's regressed here. An unexpected runtime attribute should NOT fail the workflow - it should cause a warning message but the workflow should proceed successfully